torch/_dynamo/guards.py

```diff
@@ -1902,27 +1945,125 @@ def _get_code_parts(langs):
                 langs=langs,
             )
 
-        python_code_parts, verbose_code_parts = _get_code_parts(("python", "verbose"))
-        code_parts = python_code_parts.exprs
+        if config.enable_cpp_symbolic_shape_guards:
+            # For exporting we need the python code parts
+            python_code_parts, verbose_code_parts, cpp_code_parts = _get_code_parts(
+                ("python", "verbose", "cpp")
+            )
+        else:
+            python_code_parts, verbose_code_parts = _get_code_parts(
+                ("python", "verbose")
+            )
```

Comment:
@laithsakka previously I was generating all 3 parts, `python, verbose, cpp` because it was easier to code and it made one of the compile time benchmarks fail. Now I have a condition to avoid the compile time regression. The benchmarks were useful in this case.